Sources/Workspace/DefaultPluginScriptRunner.swift

```diff
@@ -422,7 +425,11 @@ public struct DefaultPluginScriptRunner: PluginScriptRunner, Cancellable {
         // Optionally wrap the command in a sandbox, which places some limits on what it can do. In particular, it blocks network access and restricts the paths to which the plugin can make file system changes. It does allow writing to temporary directories.
         if self.enableSandbox {
             do {
-                command = try Sandbox.apply(command: command, strictness: .writableTemporaryDirectory, writableDirectories: writableDirectories + [self.cacheDir], readOnlyDirectories: readOnlyDirectories)
+                command = try Sandbox.apply(command: command,
+                                            strictness: .writableTemporaryDirectory,
+                                            writableDirectories: writableDirectories + [self.cacheDir],
+                                            readOnlyDirectories: readOnlyDirectories,
+                                            allowNetworkConnections: allowNetworkConnections)
```

Comment:
most of the recent code uses this style, I think we should make this our standard (also... run swift-format!)
```
command = try Sandbox.apply(
  command: command,
  strictness: .writableTemporaryDirectory,
  writableDirectories: writableDirectories + [self.cacheDir],
  readOnlyDirectories: readOnlyDirectories,
  allowNetworkConnections: allowNetworkConnections
)
```

Reply:
nit: `swiftformat`, not `swift-format`

Reply:
Maybe once we have a proper Xcode integration ;)